File: app/src/main/java/huce/fit/appreadstories/story/information/StoryInformationView.java
package huce.fit.appreadstories.story.information;

import huce.fit.appreadstories.model.DanhGia;
import huce.fit.appreadstories.model.Truyen;
import huce.fit.appreadstories.sqlite.Story;

public interface StoryInformationView {

    void setData(Truyen truyen);
    void setData(Story story);
    void checkInteractive(int isLike, String likeNumber);
    void followStory(int isFollow);
    void getRate(DanhGia danhGia);
    void reloadViewPaper(String text);
}

File: app/src/main/java/huce/fit/appreadstories/story/information/StoryInformationView.kt
package huce.fit.appreadstories.story.information

import huce.fit.appreadstories.model.Story

interface StoryInformationView {
    fun close()
    fun setRateStartLight(index: Int)
    fun setRateStartDark(index: Int)
    fun setData(story: Story)
    fun setRateId(rateId: Int)
    fun checkInteractive(isLike: Int, likeNumber: String)
    fun followStory(isFollow: Int)
    fun openDialogRate()
    fun addRate(ratePoint: Int, rate: String)
    fun updateRate(ratePoint: Int, rate: String)
    fun deleteRate()
    fun reloadViewPaper(text: String)
    fun checkStory()
}